### DISCUSSION TITLE
Congrats on 1k stars! 🤩✨🌟

### DISCUSSION THREAD

**Discussion by MystPi** [standardagents/arrow-js#46] [General]
I have been following ArrowJS with interest for a while now, and I just saw that the repo has reached 1k+ stars! Keep up the amazing work! 💖

**Comment by justin-schroeder** [comment]
Thank you! Let’s keep it rolling :)

**Reply by riongull** [reply]
Wow. What do you attribute the big jump to?  Curious if it was the same as how I discovered you.

**Reply by justin-schroeder** [reply]
Honestly not sure how people are hearing about Arrow - I hear there is some course that includes it or something like that? I'm curious to know!

**Reply by riongull** [reply]
I discovered you through the ui.dev newsletter about a month ago.
